build_centos.dockerfile

```diff
@@ -0,0 +1,46 @@
+# https://github.com/ghdl/ghdl/issues/1484
+# https://github.com/ghdl/ghdl/issues/1118
+
+#
+# Centos 7
+#
+
+FROM centos:centos7
+
+RUN yum update -y && yum install -y git curl bzip2 wget flex fontconfig libX11 zlib-devel
+RUN yum install -y centos-release-scl && yum install -y devtoolset-8 texinfo
+SHELL [ "/usr/bin/scl", "enable", "devtoolset-8" ]
+
+#
+# Centos 8
+#
+
+#FROM centos:centos8
+
+#RUN yum update -y && yum install -y  git curl bzip2 wget flex fontconfig libX11 zlib-devel
+#RUN yum install -y --enablerepo=powertools gcc texinfo make
+
+#
+# Common
+#
+
+WORKDIR /root
+
+RUN mkdir gnat && \
+    curl -L https://community.download.adacore.com/v1/9682e2e1f2f232ce03fe21d77b14c37a0de5649b?filename=gnat-gpl-2017-x86_64-linux-bin.tar.gz | tar -xz -C gnat --strip-components=1
+RUN cd gnat && make ins-all prefix="build"
+
+ENV PATH=/root/gnat/build/bin:$PATH
+
+RUN mkdir gcc && curl -L https://codeload.github.com/gcc-mirror/gcc/tar.gz/releases/gcc-8.3.0 | tar -xz -C gcc --strip-components=1
```

Comment:
Building and installing GHDL itself should not be done in this dockerfile. In CI, a container is created and [`ci-run.sh`](https://github.com/ghdl/ghdl/blob/master/scripts/ci-run.sh) from ghdl/ghdl is executed. See https://github.com/ghdl/docker/blob/master/run.sh#L235.

What we need here is a different dockerfile (`run_centos.dockerfile`) which contains runtime dependencies.

Reply:
Done, but I am not sure about which dependencies are for build and which for run :laughing:

Reply:
Honestly, it's a matter of trying 😆. Typical dependencies are zlib and libgnat only. For LLVM or GCC backends, you need the compiler too. For mcode you don't need a compiler, unless you want to build co-simulation modules (VPI, VHPIDIRECT...). On some platforms, you can install libgnat only, or libllvm only. On others, you need to install the full packages.

In this case, bzip2 flex fontconfig and libX11 should not be required.

Reply:
Ok. When build and run locally without problems, I will verify if something unneeded.